### DISCUSSION TITLE
Epic and Task hierarchy

### DISCUSSION THREAD

**Discussion by martinsangolt** [community/community#5882] [Other Feature Feedback, Questions, & Ideas]
In order to make Github Boards really useful as a Kanban tool — we need better Epic - Task hierarchy linking. It is really difficult to track or understand the progress of larger chunks of task with a completely flat issues structure.

**Comment by feerrenrut** [comment]
This seems to be described on the public roadmap, but without a specific timeline (only 'future'): https://github.com/github/roadmap/issues/284

See also (currently the most upvoted discussion I could find): https://github.com/github-community/community/discussions/5714
